File: Vehicules/src/main/java/org/example/microservice/vehicules/Services/UserDTO.java
package org.example.microservice.vehicules.Services;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Setter
@Getter

public class UserDTO {
    private String id;
    private String firstName;
    private String lastName;
    private String email;
    private String assignedVehicleId;
    private List<String> roles;
    public UserDTO(String id, String firstName, String lastName, String email, String assignedVehicleId, List<String> roles) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.assignedVehicleId = assignedVehicleId;
        this.roles = roles;
    }
}

File: ProduitOrder/ProduitOrder/Dto/UserDTO.cs
namespace ProduitOrder.Dto
{
    public class UserDTO
    {
        public long Id { get; set; }
        public string? Email { get; set; }
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public bool Active { get; set; }
        public List<string>? Roles { get; set; }
    }
}
